libnavigation-base/src/main/java/com/mapbox/navigation/base/formatter/DistanceFormatterOptions.kt

```diff
@@ -11,13 +10,13 @@ import java.util.Locale
  *
  * @param applicationContext from which to get localized strings from
  * @param locale the locale to use for localization of distance resources
- * @param unitType to use, or UNDEFINED to use default for locale country
+ * @param unitType to use, or null to use default for locale country
```

Comment:
This will never be `null`.

Reply:
null can be passed to use a default unit type